frontend/assets/stylesheets/layout/_layout.scss

```diff
@@ -76,6 +76,10 @@ body {
     padding-bottom: rem($gs-gutter * 2);
 }
 
+.l-align-middle {
```

Comment:
Think this should be `.u-align-middle`. I feel `.l-classes` should be reserved for larger structural helpers (e.g., applied to containers / sections). Element level helpers should be utilities.